src/components/experiment/ExperimentItem/ExperimentItem.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
+import Button from 'react-bootstrap/Button';
+
+interface ExperimentItemProps {
+  url: string
+}
+
+async function deleteExperimentFile(filename: string) {
+  await fetch(`${process.env.REACT_APP_BACKEND_ADDRESS}/experiment/${filename}`, { method: 'delete'});
+  window.location.reload(false);
+}
+
+function ExperimentItem({ url }: ExperimentItemProps): ReactElement<ExperimentItemProps> {
+  const filenameExpression = new RegExp(`${process.env.REACT_APP_BACKEND_ADDRESS}/static/experiment/(.*)`);
+  const match = filenameExpression.exec(url);
+  const filename = match ? match[1] : 'unknown filename';
+
+  return (
+    <ListGroup.Item>
+      <p>{filename}</p>
+      <Button href={url} className="mr-1">View</Button>
+      <Button onClick={() => deleteExperimentFile(filename)}>Delete</Button>
```

Comment:
I think it'd look a bit better if we had the buttons right justified on the line with the experiment name.
I'd also prefer the delete button be red.
I think we should have the delete button pop up with a modal asking for confirmation before calling the delete since deleting an experiment is a big deal

Reply:
Could we ditch the review button and just make the filename a hyperlink?

Reply:
To be honest if we're designing this UI to be as mobile-friendly as we can I think a button is nicer on button than a hyperlink.